pkg/sensors/sync.go

```diff
@@ -276,6 +276,10 @@ func UnloadSensor(ctx context.Context, bpfDir, mapDir string, sensor *Sensor) er
 		return fmt.Errorf("unload of sensor %s failed: sensor not loaded", sensor.Name)
 	}
 
+	if sensor.UnloadHook != nil {
+		sensor.UnloadHook.Unload()
```

Comment:
Here the returned error is never checked.

Reply:
True. I'll add a log output.